tests/pytests/test_acl.py

```diff
@@ -233,23 +233,21 @@ def test_acl_key_permissions_validation(env):
         ['FT.DICTADD', 'dict', 'hello'],
         ['FT.DICTDEL', 'dict', 'hello'],
         ['FT.DICTDUMP', 'dict'],
+        ['FT.ALIASADD', 'myAlias', perm_index], # Added so we can call `ALAISDEL`
         ['FT.ALIASDEL', 'myAlias'],
         ['FT.SUGADD', 'h:sug', 'hello', '1'],
         ['FT.SUGDEL', 'h:sug', 'hello'],
         ['FT.SUGGET', 'h:sug', 'hello'],
         ['FT.SUGLEN', 'h:sug']
     ]
     for command in non_index_commands:
-        try:
-            conn.execute_command(*command)
-        except Exception as e:
-            # Should not fail on permissions
-            env.assertNotContains("User does not have the required permissions", str(e))
+        env.expect(*command).noError() # We don't expect any errors
 
-    # The `test` user should also be able to access the index it has permissions
-    # to access.
     # Modify the `DROPINDEX` command that does not have the same index
     index_commands[-1][1] = perm_index
```

Comment:
We are dropping `perm_index` but using it in the last block of tests with default user.